Gem/Code/Source/MultiplayerSampleSystemComponent.h

```diff
@@ -57,7 +57,7 @@ namespace MultiplayerSample
 
         ////////////////////////////////////////////////////////////////////////
         // IMultiplayerSpawner overrides
-        Multiplayer::NetworkEntityHandle OnPlayerJoin(uint64_t userId, const Multiplayer::MultiplayerAgentDatum& agentDatum) override;
+        Multiplayer::NetworkEntityHandle OnPlayerJoin(uint64_t userId, const Multiplayer::MultiplayerAgentDatum& agentDatum, EntityPreActivationCallback preActivationCallback) override;
```

Comment:
Is the api change for OnPlayerJoin already in or another PR that this one depends on?

Reply:
This looks dependent on https://github.com/o3de/o3de/pull/10951. @AMZN-Gene can you mention that in the description?

Reply:
thank you!